CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the "vscode-sops" extension will be documented in this fi
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [0.4.1]
```

Comment:
Could you pls change the specific version to [Unreleased] instead? The semver upgrade is done when it's being released based on all changes.